.gitignore

```
@@ -5,3 +5,4 @@ node_modules
 coverage
 reports/*
 !reports/.keep
+log/*
```

```suggestion
log
```

Considering we make the `log/` directory on the fly for local dev we will want to ignore directory as well as its contents. Therefore it should be handled the same way as `build`